runtime/sam/expr/ztests/time-duration-cast.yaml

```diff
@@ -1,11 +0,0 @@
-zed: cut t:=time(t),d:=duration(d)
-
-input: |
-  {t:"1",d:"1"}
-  {t:1,d:1}
```

Comment:
runtime/ztests/expr/cast/duration.yaml doesn't include this case (casting an integer) so I think we should add it over there.